### PR TITLE
[bugfix] add standalone and format to "cs" locale

### LINKED ISSUE #5074: No formatted month for 'cs' locale
There are no formatted month for CS locale as for RU e.g.:

```
    var ru = moment.defineLocale('ru', {
        months : {
            format: 'января_февраля_марта_апреля_мая_июня_июля_августа_сентября_октября_ноября_декабря'.split('_'),
            standalone: 'январь_февраль_март_апрель_май_июнь_июль_август_сентябрь_октябрь_ноябрь_декабрь'.split('_')
        },...
```

### PR DESCRIPTION
**Pull request by Kartik18g**
fixes #5074 
I've updated the "cs" locale by implementing standalone and formats, I was confused why these changes were needed because cs locale code was able to handle inflections like `leden - ledna`. `únor - února` `březen-března` etc without any problems. Then I realized adding format and standalone to 'cs' locale makes the code consistent with other locales like 'hi' and 'ru' .
After making changes to the "cs" locale it's test needed to be updated because of moment expecting standalone values, I also added 'format month case test' for accusations and nominations.

Now, for the most part, cs locale code is consistent with other locales, but if you feel something needs to be changed I can do that.

### CODE REVIEW

**Comment by marwahaha** [comment]
@petrbela could you take a look?

File: src/test/locale/cs.js [hunk 1]
```diff
@@ -133,6 +133,30 @@ test('format month', function (assert) {
     }
 });
 
+test('format month case', function (assert) {
+    var months = {
+            nominative: 'ledna_února_března_dubna_května_června_července_srpena_září_říjena_listopada_prosince'.split(
```
Comment (petrbela) [inline src/test/locale/cs.js:138]:
```suggestion
            nominative: 'ledna_února_března_dubna_května_června_července_srpna_září_října_listopadu_prosince'.split(
```

**Review by petrbela — CHANGES_REQUESTED** [review]
The implementation looks good, just added the correct Czech inflections in the comments

**Comment by Kartik18g** [comment]
@petrbela Thanks for providing correct inflections, i've updated them in the latest commit. All tests passed successfully everything looks good, ready to be merged .

**Comment by Kartik18g** [comment]
@petrbela  can you take a look at it?

File: src/locale/cs.js [hunk 2]
```diff
@@ -90,7 +95,7 @@ export default moment.defineLocale('cs', {
     monthsShortRegex: monthsRegex,
     // NOTE: 'červen' is substring of 'červenec'; therefore 'červenec' must precede 'červen' in the regex to be fully matched.
     // Otherwise parser matches '1. červenec' as '1. červen' + 'ec'.
-    monthsStrictRegex: /^(leden|ledna|února|únor|březen|března|duben|dubna|květen|května|červenec|července|červen|června|srpen|srpna|září|říjen|října|listopadu|listopad|prosinec|prosince)/i,
+    monthsStrictRegex: /^(leden|ledna|února|únor|březen|března|duben|dubna|květen|května|červenec|července|červen|června|srpena|srpna|srpen|září|říjena|říjen|října|listopada|listopadu|listopad|prosinec|prosince)/i,
```
[the remark this answers is not in the document]
Comment (petrbela) [reply src/locale/cs.js:98]: Can you also revert this line pls?

**Review by petrbela — CHANGES_REQUESTED** [review]

**Comment by Kartik18g** [comment]
@petrbela as you said I've reverted "monthsStrictRegex" to its original value.

**Review by petrbela — APPROVED** [review]

**Comment by indexxd** [comment]
@marwahaha the inflections here are wrong, i have created a [PR](https://github.com/moment/moment/pull/6063) to fix this 

@petrbela could you please take a look at this?

**Pull request merged**